src/server/routes/auth.js

```diff
@@ -55,4 +57,68 @@ router.get('/logout', (req, res) => {
 	res.redirect(status.SEE_OTHER, '/');
 });
 
+async function deleteUserAndGetResCode(orm, metabrainzUserID, accessToken) {
+	if (!_.isString(accessToken)) {
+		return status.BAD_REQUEST;
+	}
+
+	const USER_INFO_URL = 'https://musicbrainz.org/oauth2/userinfo';
```

Comment:
It'd be nice if the MusicBrainz server URL were configurable...I had to hand-edit this file and a couple in the `passport-musicbrainz-oauth2` module so I could test things against my local MB instance.